src/lib/config/homestead.json

```diff
@@ -14,6 +14,7 @@
   "explorer": "https://etherscan.io",
   "explorerName": "Etherscan",
   "subgraph": "https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2",
+  "balancerApi": "https://berfqtb52bc4hetoouojoshqem.appsync-api.eu-west-2.amazonaws.com/graphql",
```

Comment:
Can we get api.balancer.fi/graphql setup?

Reply:
Yea will get that setup next.